variables.tf

```diff
@@ -178,6 +178,18 @@ variable "key_name" {
   default     = null
 }
 
+variable "custom_user_data" {
```

Comment:
you shouldn't need the additional boolean variable, you should be able to just check the one below

Reply:
Latest commit has removed this variable